src/components/CompaniesGrid/style.module.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(330px, 1fr));
+  margin-top: 70px;
```

Comment:
Nitpick but let's stick to rem units in general everywhere (1rem = 16px) for better page responsiveness on zooming in/out